### ISSUE TITLE
Manage <br /> for HTML TextStyling with docx and odt

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#130]
```
Manage <br /> for HTML TextStyling with docx and odt.
```

Original issue reported on code.google.com by `angelo.z...@gmail.com` on 8 Jul 2012 at 8:29

**Comment by GoogleCodeExporter** [comment]
```
Done. This feature will be available on 0.9.8. JUnit was developped :

* for docx : see testLineBreak test at 
http://code.google.com/p/xdocreport/source/browse/document/fr.opensagres.xdocrep
ort.document.docx/src/test/java/fr/opensagres/xdocreport/document/docx/textstyli
ng/DocxDocumentHandlerTestCase.java
* for odt : see testLineBreak test at 
http://code.google.com/p/xdocreport/source/browse/document/fr.opensagres.xdocrep
ort.document.odt/src/test/java/fr/opensagres/xdocreport/document/odt/textstyling
/ODTDocumentHandlerTestCase.java
```

Original comment by `angelo.z...@gmail.com` on 9 Jul 2012 at 9:22
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
The new version works fine with new line <br>! Thanks a lot! Regards Alexander
```

Original comment by `googlec...@norz.de` on 20 Jul 2012 at 12:33
